File: TabLayoutJava/app/src/main/java/fr/busin/tablayout/SecondFragment.java
package fr.busin.tablayout;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

public class SecondFragment extends Fragment {

    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {

        View root = inflater.inflate(R.layout.fragment_second, container, false);
        final TextView textView = root.findViewById(R.id.section_label);

        textView.setText("Second Fragment");

        return root;
    }
}

File: TabLayoutKotlin/app/src/main/java/fr/busin/tablayout/SecondFragment.kt
package fr.busin.tablayout

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.fragment.app.Fragment
import fr.busin.tablayout.R

class SecondFragment : Fragment() {

    override fun onCreateView(
            inflater: LayoutInflater, container: ViewGroup?,
            savedInstanceState: Bundle?
    ): View? {
        val root = inflater.inflate(R.layout.fragment_second, container, false)

        val textView: TextView = root.findViewById(R.id.section_label)
        textView.text = "Second Fragment"

        return root
    }

}